src/modules/ROOT/pages/changelog/202303/05.json

```diff
@@ -100,23 +100,23 @@
     },
     {
       "what": [
-        "conditional field being displayed even if its condition says the opposite."
+        "conditional fields being displayed when viewing records even though their condition evaluates to `false`."
       ],
       "refs": [
         "https://github.com/cortezaproject/corteza/commit/e081faab821509d7c3f4efbbf707b872e923d08c"
       ]
     },
     {
       "what": [
-        "flow for creating a record in a modal. Previously, the user wasn't able to create a record in the modal because the page for creating a record was opened behind the modal. Now, the same view opens in the modal and the user can create a record from there."
+        "not being able to create a record in a modal because the view opens behind the modal."
```

Comment:
Looks good but I'd incorporate this bit also `Now, the same view opens in the modal and the user can create a record from there.`. Perhaps under `how`?